airflow/www/views.py

```diff
@@ -603,13 +603,25 @@ def dag_details(self, session=None):
     def rendered(self):
         dag_id = request.args.get('dag_id')
         task_id = request.args.get('task_id')
+
+        dag = dagbag.get_dag(dag_id)
```

Comment:
Can you change the order of these and next checks? We should not pass parameters to other classes if they don't have the correct format.